source/install/config-mattermost-server.rst

```diff
@@ -30,4 +30,6 @@ Configuring Mattermost Server
 7. Feel free to modify other settings.
 8. Restart the Mattermost Service.
 
-  ``sudo restart mattermost``
+  On Ubuntu 14.04 and RHEL 6.6: ``sudo service mattermost restart``
```

Comment:
Oh I see what you mean, because the init script isn't also included in Upstart, `service` wouldn't be used. I can revert  this back.